src/stores/streamlit-data.ts

```diff
@@ -15,8 +15,8 @@ export const useStreamlitDataStore = defineStore('streamlit-data', {
       return this.args.components
     },
     allDataForDrawing: (state) => state.dataForDrawing,
-    sequenceData: (state): SequenceData | undefined =>
-      state.dataForDrawing.sequence_data as unknown as SequenceData | undefined,
+    sequenceData: (state): SequenceDataDictionary | undefined =>
```

Comment:
I think it will have to be adapted but change could be minimal as FLASHViewers `SequenceView`only uses a single sequence. Thus, the sequence with ID zero (or similar) could be used in all cases.